src/request.ts

```diff
@@ -336,13 +330,12 @@ class DatastoreRequest {
    *   const apiResponse = data[0];
    * });
    */
-  delete(keys, gaxOptions?, callback?) {
-    if (is.fn(gaxOptions)) {
-      callback = gaxOptions;
-      gaxOptions = {};
-    }
-
-    callback = callback || (() => {});
+  delete(keys:Entities): void;
```

Comment:
That doesn't seem right.  If no callback is provided, it should return a Promise<WhatEverType> here.

Reply:
Agreed - another lint bypass for `[ts] Overload signature is not compatible with function implementation. [2394]` :) Changed!